react/ReleasesPage.tsx

```diff
@@ -49,7 +49,7 @@ class ReleasesPage extends Component<{} & ProfileData, ReleasesPageState> {
       profile: { profile },
     } = this.props
 
-    const firstName = profile ? profile.name.split(' ')[0] : ''
+    const firstName = profile && profile.name ? profile.name.split(' ')[0] : ''
```

Comment:
I think you can do that already
```suggestion
    const firstName = profile?.name?.split(' ')[0] || ''
```

Reply:
:notbad: